simple_generator.py

```diff
@@ -0,0 +1,191 @@
+
+from midiutil.MidiFile import MIDIFile
+import random
+import sys
+import pygame
+
+
+# we expect input mode
+
+# work with input arguments
+arr = sys.argv[:]
+if len(arr) > 1:
+    file = arr[1]
+else:
+    file = "out.mid"
+
+
+# choose working mode
+try:
+    mode = input("""Press 
+P - play 
+S - safe 
+B - both
+""")
+
+except EOFError:
+    print("Обработали исключение EOFError")
+
+
+# 1 here is number of tracks
+MyMIDI = MIDIFile(1)
+
+# Tracks are numbered from zero
+# Times are measured in beats
+track = 0
+time = 0
+
+MyMIDI.addTrackName(track, time, "Sample Track")
+MyMIDI.addTempo(track, time, 120)
+random.seed()
+binfile = open(file, 'wb')
+
+
+def make_music_rand():
+    seq = []
+    seq_ac = []
+    i = 0
+    while i < 99:
+        a = random.randint(1, 7)
+        seq.append(a)
+        i = i + 1
+
+    i = 0
+    while i < 99:
+        a = random.randint(1, 7)
+        seq_ac.append(a)
+        i = i + 3
+    write_sequence_ac(seq_ac)
+    return seq
+
+
+# now we expect list here
+def write_sequence(args):
```

Comment:
Literally the worst argument name. It makes you think that this is `*args` variable arguments, but it's not!
After that, it's not describing anything

Reply:
fixed